src/docs/product/performance/index.mdx

```diff
@@ -45,3 +45,17 @@ Below the graphs are a list of transactions that may be filtered down based on g
 ## Key Transaction View
 
 If you have transactions you frequently return to; you can mark each one as a key transaction in their corresponding [summary views](/product/performance/transaction-summary/#key-transaction). This enables a shortcut to view only Key Transactions by toggling the tab in the upper right corner of the homepage.
+
+## Trends View
+
+Trends surfaces transactions that have had significant changes in their performance over time. This view can be found by toggling the tab in the upper right corner of the homepage.
```

Comment:
Unsure about `This view can be found by toggling the tab in the upper right corner of the homepage.` People might get confused with `homepage`.

`You can find it by going to the top right corner beside Transactions and Key Transactions.`